packages/client/src/lib.js

```diff
@@ -43,9 +50,28 @@ const MAX_CHUNK_SIZE = 104857600
 /** @typedef { import('./lib/interface.js').CIDString} CIDString */
 /** @typedef { import('./lib/interface.js').PutOptions} PutOptions */
 /** @typedef { import('./lib/interface.js').PutCarOptions} PutCarOptions */
+/** @typedef { import('./lib/interface.js').RateLimiter } RateLimiter */
 /** @typedef { import('./lib/interface.js').UnixFSEntry} UnixFSEntry */
 /** @typedef { import('./lib/interface.js').Web3Response} Web3Response */
 
+/**
+ * @returns {RateLimiter}
+ */
+export function createRateLimiter (
+  requestsLimit = RATE_LIMIT_REQUESTS,
+  timePeriod = RATE_LIMIT_PERIOD
```

Comment:
I did not expose these in the public API as users should not change them. They cannot give themselves more requests!

Reply:
I made these into parameters so that I can override them in the test. Otherwise the test has to take 30 seconds, which is sloooooow. (Users of the public API could override these defaults if they wanted to, but it would only be to their own detriment, as CloudFlare will then punish them with a 30 penalty when they exceed the limit!)

Reply:
Sorry, 10 seconds, but it still seems like a lot longer than we want a single test to take.

Reply:
We shouldn't change our public API to accommodate tests, and I think it could be misleading to users to allow these parameters to be changed. IMHO we should either take the hit and wait 10 seconds for the test or move this into a separate file that is not exposed publically and call that function from here (and use the non-public file in the tests).

Reply:
Ok, I've refactored this, so the public API no longer has those parameters. For the test of the rate limiter itself, we take the 10 second hit, but for the `'adds big files'` test (which is where the real slowness was) we now pass in a mock ratelimiter to avoid being throttled.

Ready for re-review.